docs/gravity-bridge/systemd/setup-gorc-service.sh

```diff
@@ -54,7 +54,7 @@ check_gorc_setup() {
 }
 
 download_service() {
-    curl -s https://raw.githubusercontent.com/mofhusseini/cronos/master/docs/gravity-bridge/systemd/gorc.service.template -o $BASEDIR/gorc.service.template
+    curl -s https://raw.githubusercontent.com/crypto-org-chain/cronos/main/docs/gravity-bridge/systemd/gorc.service.template -o $BASEDIR/gorc.service.template
```

Comment:
using main is fine for now but better to use a tag/version/hash after release

Reply:
good point